extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -410,6 +363,13 @@ export const AzureProvisionDialog: React.FC = () => {
     }
   };
 
+  useEffect(() => {
+    if (token) {
+      getSubscriptions(token).then(setSubscriptions);
```

Comment:
make sure the component is not unmounted when the setState is called.

Reply:
@hatpick How do I do that with hooks? (I pulled this code directly from another existing component)